contents/python/introduction-to-python.md

```diff
@@ -12,6 +12,21 @@ For absolute beginners, check out [this video](https://www.youtube.com/watch?v=N
 
 Both newbies and experienced programmers can also benefit from [The Python Tutorial](https://docs.python.org/3/tutorial/index.html), which aims to introduce readers to Python's unique features and style.
 
+## Python 2 vs Python 3
```

Comment:
I'd argue that as an introductory piece, it would be best to recommend one version to the reader. Without establishing a recommended version, it would be hard to introduce Python meaningfully without confusing the reader.

Personally, I would recommend Python 3 for beginners ([this stance is also supported by a core member of Python](http://python-notes.curiousefficiency.org/en/latest/python3/questions_and_answers.html#why-is-python-3-considered-a-better-language-to-teach-beginning-programmers)). It's been around [**10 years** since Python 3 debuted](https://www.python.org/download/releases/3.0/), so many popular libraries support it these days, and most Linux distributions have Python 3.x already installed. Most reasons that tutorials used to give about sticking with Python 2 are no longer relevant, or not as significant. You would want to learn Python 2, if you're working with legacy systems, or with a company that still uses Python 2, of course.

However, even if you're aiming to learn Python 2, as a complete beginner it's still recommended to learn Python 3 first. [As the official Python wiki mentions](https://wiki.python.org/moin/Python2orPython3/#Which_version_should_I_use.3F), 

> "instructors introducing Python to new programmers should consider teaching Python 3 first and then introducing the differences in Python 2 afterwards (if necessary), since Python 3 eliminates many quirks that can unnecessarily trip up beginning programmers trying to learn Python 2."

The first link in my comment also brings up quirks in Python 2, that are resolved in Python 3, that make learning Python 3 more consistent for beginners.